### PR TITLE
LTP: Add possibility to run ver_linux also at the end of testing

### PR DESCRIPTION
**Pull request by pevik**
Some tests causes loading kernel modules on demand, some loads them
deliberatelly. Thus it can be interesting to see which modules were
added after testing. The easiest way is to do it with ver_linux script.

Activate with LTP_RUN_VER_LINUX_TWICE=1

Verification run: http://quasar.suse.cz/tests/1111

### CODE REVIEW

**Comment by pevik** [comment]
@richiejp I didn't want to have ver_linux output if not needed, but maybe running it twice for all tests is ok.

File: tests/kernel/run_ltp.pm [hunk 1]
```diff
@@ -492,6 +494,10 @@ EOF
 
     script_run('[ "$ENABLE_WICKED" ] && systemctl enable wicked');
     script_run('journalctl --no-pager -p warning');
+
+    if ($run_ver_linux_twice) {
+        script_output($ver_linux_path);
```
Comment (asmorodskyi) [inline tests/kernel/run_ltp.pm]: if you don't care about return value use "script_run" instead

**Comment by pevik** [comment]
@richiejp 

**UPDATE:**
LTP: Run ver_linux also at the end of testing
+ Safe output of both running into a file (better for comparison).
+ Update copyright.

File: tests/kernel/run_ltp.pm [hunk 2]
```diff
@@ -348,7 +348,11 @@ sub run {
 
     assert_script_run('cd $LTPROOT/testcases/bin');
 
-    my $ver_linux   = script_output('$LTPROOT/ver_linux');
+    my $ver_linux = '$LTPROOT/ver_linux';
+    my $log       = '/tmp/ver_linux_before.txt';
+    script_run("$ver_linux > $log 2>&1");
+    upload_logs($log);
+    my $out         = script_output("cat $log");
```
Comment (asmorodskyi) [inline tests/kernel/run_ltp.pm]: out - is too broad name . What about `$ver_linux_out` ?

Comment (okurz) [reply tests/kernel/run_ltp.pm]: IMHO it's fine.

Comment (okurz) [reply tests/kernel/run_ltp.pm]: Hm, ok. After reviewing how it's used I agree with @asmorodskyi :)

Comment (pevik) [reply tests/kernel/run_ltp.pm]: @asmorodskyi @okurz `$log` is also broad name :-). I changed all 3 variables (I also don't like broad names, but `$ver_linux_out`, `$ver_linux_log`, `$ver_linux_path` looked quite long to me).

**Pull request merged**